hera_pspec/utils.py

```diff
@@ -1333,7 +1333,7 @@ def uvd_to_Tsys(uvd, beam, Tsys_outfile=None):
 
     return uvd
 
-def uvp_noise_error(uvp, auto_Tsys=None, err_type='P_N', precomp_P_N=None, P_SN_correction=True,  num_steps_scalar=2000, little_h=True):
+def uvp_noise_error(uvp, auto_Tsys=None, auto_Tsys_int=None, err_type='P_N', precomp_P_N=None, P_SN_correction=True,  num_steps_scalar=2000, little_h=True):
```

Comment:
Why the new kwarg? Needs a docstring too.

Reply:
I think this is an artifact of an idea that I never ended up following through on. I don't remember what it is and it doesn't look like this kwarg is called anywhere else in hera_pspec so I've removed it.